src/server/raft_client.rs

```diff
@@ -83,23 +84,22 @@ impl Conn {
                 (batch_msgs, WriteFlags::default().buffer_hint(false))
             }))
             .then(move |r| {
-                let fallback = match r {
-                    Err(GrpcError::RpcFinished(Some(RpcStatus { status, .. })))
-                        if status == RpcStatusCode::UNIMPLEMENTED =>
-                    {
-                        true
+                let mut fallback = false;
+                if let Err(e) = r {
+                    warn!("batch_raft sink error"; "to_addr" => &batch_addr, "err" => ?e);
```

Comment:
Can two errors be merged just like before?

Reply:
So error from `BatchSink` will be an option passed into `or_else`, and error from `Sink` in `or_else` is an another option. It will make it look more complex. So I sperate them, one for sink and one for receiver.